docs/manual/subtyping-checker.tex

```diff
@@ -8,9 +8,7 @@
 
 The Subtyping Checker can accommodate all of the type system enhancements that
 can be declaratively specified (see Chapter~\ref{creating-a-checker}).
-This includes type introduction rules (implicit
-annotations, e.g., literals are implicitly considered \refqualclass{checker/nullness/qual}{NonNull}) via
-the \refqualclass{framework/qual}{ImplicitFor} meta-annotation, and other features such as
+This includes type introduction rules meta-annotation, and other features such as
```

Comment:
I think it sounds better as `rule meta-annotations`, doesn't it?
Wouldn't a reference to `QualifierForLiterals` still be useful here?